charts/coredns/values.yaml

```diff
@@ -87,8 +87,10 @@ isClusterService: true
 priorityClassName: ""
 
 # Configure the pod level securityContext.
-podSecurityContext: {}
-
+podSecurityContext:
+  sysctls:
+  - name: net.ipv4.ip_unprivileged_port_start
+    value: "53"
```

Comment:
This should already be covered by https://github.com/coredns/helm/blob/master/charts/coredns/values.yaml#L94C1-L97C25

Reply:
This is an alter way and  with the new sysctl, we can remove the `NET_BIND_SERVICE` capability.
```
          capabilities:
            add:
            - NET_BIND_SERVICE
            drop:
            - ALL
```

Reply:
Since we already have a solution in place for this, I don't think we should make any changes, this works as intended and there's no real value in replacing the `capabilities` with the `sysctls`